docs/docfx.json

```diff
@@ -15,6 +15,7 @@
             "src": "../src"
         }],
         "dest": "api",
+        "filter": "filterConfig.yml",
```

Comment:
All APIs which marked with `InternalApiAttribute` will be removed from API referenced

Reply:
Nice